package.json

```diff
@@ -109,6 +109,9 @@
     "type": "git",
     "url": "https://github.com/nodejs/nodejs.dev"
   },
+  "resolutions": {
+    "sharp": "0.27.2"
+  },
```

Comment:
I was having a lot of trouble running locally, specifically because of the `sharp` dependency shared between `gatsby-plugin-manifest` and `gatsby-plugin-sharp`. One would get the wrong version and I couldnt run the project locally. I added this so installing will force the correct version of `sharp`

Reply:
Which version of node are you using? Are you using npm or yarn? 
i haven’t see this issue before. Have you tried using the latest Node LTS? 14.x

Reply:
I tried both 10.x when I run `nvm use` and the latest Node LTS. Before I added this, the `gatsby-plugin-sharp` was downloading sharp version `0.25.6` which was incompatible. I also tried nuking package lock and node modules to no avail. 

One thing I will say is that I didnt see this issue at all after upgrading gatsby and all the associated plugins to 3.x, so if this is merged I think we can remove it in your upgrade PR [here](https://github.com/nodejs/nodejs.dev/pull/1166)